index.rst

```diff
@@ -352,28 +385,145 @@ To make sure the system would work properly we had to enable two additional feat
+
+  .. code-block:: text
+
+    ========================================================================================
+    Report      : INFO
+    Date        : 2020-09-03T17:19:02+0000
+    Description : Missed heartbeat for node 105060556. (DCPSPublication is complete)
+    Node        : ospl-daemon-m482p
+    Process     : spliced <148347>
+    Thread      : OSPL Garbage Collector 7f48013ae700
+    Internals   : P674/6.10.4/de6011b/2a9a655/v_spliced/v_spliced.c/2034/769/1599153542.755942957/2
+    ========================================================================================
+    Report      : WARNING
+    Date        : 2020-09-03T17:40:49+0000
+    Description : writer 6672e859:7b:1:a02 topic d_sampleChain waiting on high watermark due to reader 4254ba5d:7f:1:1507
+    Node        : ospl-daemon-m482p
+    Process     : ddsi2 <148361>
+    Thread      : xmit.user 7f20e331b700
+    Internals   : P674/6.10.4/de6011b/2a9a655/ddsi2/q_transmit.c/923/0/1599154849.155224251/2
+    ========================================================================================
+
+In order to solve this issue, it required some further understanding of how DDS communication works and more study on the daemons configuration.
+The goal was to be able to increase the resources available for the OpenSplice daemons to handle the increased load while also reducing the load on the DDS services.
+
+.. _Increasing-OpenSplice-daemon-resources:
+
+Increasing OpenSplice daemon resources
+--------------------------------------
+
+The OpenSplice DDS software configuration provides a couple of different ways of increasing the computing resources available for the daemons, mostly revolving around the allotted memory.
+Considering that the :ref:`initial configuration <Appendix-Initial-Shared-Memory-OpenSplice-Configuration>` for the daemons allocated 10 times as much shared memory space then the `standard <https://istkb.adlinktech.com/article/vortex-opensplice-dds-configuration/>`__, we also increased by 10 times the amount of memory available for other daemon services.
+
+In addition to increasing resources for the daemons we also enabled two new features that helps reduce system load; squash participants and lazy alignment.
+
+The squash participant feature allow daemons to "merge" the data structure of all the applications attached to it into a single data structure.
+Therefore, instead of presenting several "participants" to the network, the daemon presents a single unified system that is much leaner to maintain.
+This feature no only helps reduce the data structure complexity but is also reduces discovery time for new components in the system.
+The drawback to squashing participants is that we loose the ability to inspect individual applications data structure.
```

Comment:
Perhaps "the data structures for individual applications".

I confess I don't really understand what data structures these are. If this is the DDS topic cache it suggests that we can't tell which application wrote which data, which seems harmless since we only have one writer for each topic (except for indexed components, and even then we only have one writer per index).